en/identity-server/7.0.0/docs/apis/scim2/scim2-batch-operations.md

```diff
@@ -4,6 +4,14 @@ Follow the topics given below to understand how **Bulk** operations can be used
  
 The SCIM2 API allows you to send multiple resource operations in a single request. That is, you can add new records (POST data), replace an existing record (PUT data), update elements of an existing record (PATCH data), and delete records (DELETE data) in bulk. These bulk operations are supported for managing users and groups with the SCIM API in WSO2 Identity Server.
  
+!!! Info
+    The default limit for bulk operations is set to 1,000 operations per request. To modify this limit, add the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
+
+    ```
+    [scim2]
+    max_bulk_operations = 500
+    ```
+    
```

Comment:
It's a bit unclear about the upper margin of the `max_bulk_operations` configuration. When an administrator has to import a large number of users, they want to know the maximum number of users that can be imported in a single bulk operation. Can they go beyond the default limit of 1000 and set a higher value? Better to mention the maximum limit that can be set for `max_bulk_operations` config

Reply:
Yes, they can exceed 1000, but we cannot specify an exact upper limit in the documentation, as it depends on the customer's deployment configurations, database type, and network conditions.